pkg/agent/config.go

```diff
@@ -136,6 +136,8 @@ type Config struct {
 	KafkaSASLClientSecretPath string `env:"KAFKA_SASL_CLIENT_SECRET_PATH"`
 	// ProfilePort sets the listening port for Go's Pprof tool. If it is not set, profile is disabled
 	ProfilePort int `env:"PROFILE_PORT"`
+	// Enable RTT calculations for the flows, default is 0 (disabled), set to 1 to enable.
```

Comment:
`default is false set to true to enable`

Reply:
Fixed.